dependency-management/dependency-management-impl/src/main/java/io/cloudslang/dependency/impl/services/DependencyServiceImpl.java

```diff
@@ -200,6 +233,37 @@ private void buildDependencyFile(String[] gav) {
             throw new IllegalStateException(fileToReturn.getPath() + " not found");
         }
         appendSelfToPathFile(gav, fileToReturn);
+        sendMavenDependencyBuildFinishedEvent(gav);
+    }
+
+    private void sendMavenDependencyBuildFinishedEvent(String[] gav) {
+        String message = "Artifact download complete";
+        if (gav.length >= 3) {
+            message = String.format("Downloading complete for artifact with gav: %s:%s:%s ", gav[0], gav[1], gav[2]);
+        }

```

Comment:
it is a weird condition `gav.length < 3` maybe we should at least log if not fail